specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,39 +15,36 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
```

Comment:
nit: are we usually deleting the template section comments in early VEP stage? we may keep those to help with populating the VEP further